helm-charts/falcon-sensor/templates/NOTES.txt

```diff
@@ -1 +1,5 @@
-1. Get the application URL by running these commands:
+The CrowdStrike Falcon sensor should be spinning up on all your kubernetes nodes
+now. There should be no further action on your part unless no Falcon Sensor
+container exists in your regisry. If you forgot to add a Falcon Sensor image to
+your image registry before you ran `helm install`, please add the Falcon Sensor
+now; otherwise, your deployment will fail until there is a valid image to pull.
```

Comment:
Is there a specific name of the container that the helm chart will look for, eg falcon-container? May want to call that out to clarify error message.